README.adoc

```diff
@@ -20,33 +20,46 @@ https://github.com/gosecure/malboxes
 
 == Requirements
 
-* Python 3
+* Python 3.3+
+* appdirs
 * jinja2
-* packer (sometimes called packer-io)
-* vagrant
+* packer: https://www.packer.io/intro/getting-started/setup.html
+* vagrant: https://www.vagrantup.com/downloads.html
+
+== Installation
+
+    sudo pip install git+https://github.com/GoSecure/malboxes.git@pip-packaging#egg=malboxes
```

Comment:
Have you tried that with a machine that didn't have the dependencies ? It doesn't work for me and the traceback indicates that it cannot import ```appdirs``` (so that should be the same for all our dependencies) because ```__version__``` is imported from ```__init__.py``` which tries to import the dependencies. I got the same from my server too. Also, if we are py3, shouldn't that be ```pip3``` since pip defaults to ```pip2``` in lots of distros (```pip --version``` points to py2.7 for both of the machines I tested on) ? [Official docs](http://python-packaging.readthedocs.io/en/latest/minimal.html#creating-the-scaffolding) doesn't seem to discourage hardcoding the version so couldn't we do that ? [Django](https://github.com/django/django/blob/master/django/__init__.py) does something interesting that doesn't involve hardcoding so that would be an option too.